netbox/dcim/views.py

```diff
@@ -341,8 +342,21 @@ def get(self, request):
             'reverse': reverse,
             'rack_face': rack_face,
             'filter_form': forms.RackElevationFilterForm(request.GET),
+            'show_images': request.user.config.get('rack_elevation.show_images', False)
```

Comment:
Images should be enabled by default to preserve the existing behavior.

Reply:
It will be easy to change